lib/peep.rb

```diff
@@ -0,0 +1,31 @@
+require 'pg'
+
+class Peep
+  attr_reader :message
+
+  def initialize(message)
+    @message = message
+  end
+
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
+    
+    result = connection.exec('SELECT * FROM peeps')
+    result.map { |peep| p peep['message'] }
+  end
+
+  # Initial, hard-coded version
+
+  # def self.all
+  #   # [
+  #   #   "Hello, this is a new peep!", 
+  #   #   "What peep shall I write today?",
+  #   #   "My favourite food is pizza"
+  #   # ]
+  # end
```

Comment:
For the purposes of this exercise, it's helpful that you've left this in, so I can see how it's progressed from the hard-coded version. Aside from this exercise, I imagine you can delete it, once the other version is functioning.

Reply:
Yes I left it in for me - to remind myself that the output of the query at lines 17 and 18 should be an array. But it's a good reminder that I should take it out in a "final" version.